packages/components/e2e/component-objects/src/main/java/org/talend/component/AppHeaderBar.java

```diff
@@ -12,9 +12,9 @@
  */
 public class AppHeaderBar extends Component {
 
-    private static final String NAME = "HeaderBar";
+    private static final String HEADER_BAR_NAME = "HeaderBar";
```

Comment:
No need to repeat `HEADER_BAR` since you are already inside the `AppHeaderBar` class.

Reply:
i will discard my PR beacause this doen't have any interest but i am not sure that variable shadowing in this case it's a good practice